app/graph/types/media_type.rb

```diff
@@ -35,61 +35,14 @@
     }
   end
 
-  connection :annotations, -> { AnnotationType.connection_type } do
-    argument :context_id, types.Int
-
-    resolve ->(media, args, ctx) {
-      media.cached_annotations(annotation_types, media_context(media, args, ctx))
-    }
-  end
-
-  field :annotations_count do
+  field :pm_dbid do
```

Comment:
I called pm_dbid with context_project  `media.pm_dbid(ctx[:context_project])` to return ProjectMediaId as redirect URL format is `project/id/media/pm_id` and should remove this method after #5587 implementation